### PR TITLE
fix(DB/SAI) Adds missing (36332) Rake ability to (17350) Royal Blue Flutterer

### PR DESCRIPTION
**Pull request by wdraenor**
Adds missing (36332) Rake ability to (17350) Royal Blue Flutterer

## Changes Proposed:
-  Adds missing (36332) Rake ability to (17350) Royal Blue Flutterer
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/16896

## SOURCE:

https://www.wowhead.com/tbc/npc=17350/royal-blue-flutterer#abilities
Video of Royal Blue Flutterer using Rake ability: https://www.youtube.com/watch?v=r0pZaRBga3k&t=22

## Tests Performed:

- Linux Mint 21.1
- I fought Royal Blue Flutterers and they used their Rake ability in combat like they should.

## How to Test the Changes:

1. .go xyz -1453.013672 -11623.257812 20.930595 530 4.382232
2. fight royal blue flutterers
3. realize they are now using rake ability 

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1691257289493032818.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 17350;
+
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 17350);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1691257289493032818.sql:3]: don't forget the backticks :)

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1691257289493032818.sql:3]:
```suggestion
DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 17350);
```

Other than that I think this looks good =)

Comment (wdraenor) [reply data/sql/updates/pending_db_world/rev_1691257289493032818.sql]: Thanks, I'll remember that for next time :)

**Review by Kitzunu — APPROVED** [review]

**Comment by elthehablo** [comment]
`- [ ]` needs to be used as follows `- [ ]` = task not completed `-[x]` = task completed.

See:
- [ ] task1
- [x] task2

Otherwise good job! Gz on first PR. How did you get the timers?

**Comment by wdraenor** [comment]
> `- [ ]` needs to be used as follows `- [ ]` = task not completed `-[x]` = task completed.
> 
> See:
> 
>     * [ ]  task1
> 
>     * [x]  task2
> 
> 
> Otherwise good job! Gz on first PR. How did you get the timers?

I fixed the TODO list. I got the timers from the (17349) Blue Flutterers. It's likely that the spell timers would be the same for the Royal Blue Flutterers as well, based on gameplay and the video.

**Pull request merged**